l10n_it_fatturapa_out/models/partner.py

```diff
@@ -16,7 +16,8 @@ class ResPartner(models.Model):
 
     @api.constrains("max_invoice_in_xml")
     def _validate_max_invoice_in_xml(self):
-        if self.max_invoice_in_xml < 0:
-            raise ValidationError(
-                _("The max number of invoice to group can't be negative")
-            )
+        for partner in self:
+            if partner.max_invoice_in_xml < 0:
+                raise ValidationError(
+                    _("The max number of invoice to group can't be negative")
```

Comment:
A questo punto aggiungerei un riferimento al partner nel messaggio di errore, così l'utente sa qual è il partner che fallisce.
Grazie

Reply:
Ok. In questo caso, visto che ci sono i file .po per tutte le lingue, mi tocca aggiornare tutti i file ? che strumento posso usare?

Reply:
> Ok. In questo caso, visto che ci sono i file .po per tutte le lingue, mi tocca aggiornare tutti i file ? che strumento posso usare?

Non serve che rigeneri i .po per le lingue, se ne occuperà il sistema.

P.S.: per completezza nel caso tu voglia fare test locali. Per aggiornare un .po partendo dal .pot aggiornato (estratto da Odoo) puoi utilizzare `msgmerge`.

Es.
`msgmerge -U <cod_lingua>.po <nome_modulo>.pot`

Reply:
Grazie! 
Ho aggiunto nome del partner nel messaggio di errore.

Serve altro ?